test/plugin/test_out_copy.rb

```diff
@@ -217,5 +217,37 @@ def test_ignore_error
       end
     end
   end
+
+  IGNORE_IF_PREV_SUCCESS_CONFIG = %[
+    <store ignore_error>
+      @type test
+      name c0
+    </store>
+    <store ignore_if_prev_success ignore_error>
+      @type test
+      name c1
+    </store>
+    <store ignore_if_prev_success>
+      @type test
+      name c2
+    </store>
+  ]
+
+  def test_ignore_if_prev_success
```

Comment:
Does this code test `ignore_if_prev_success` feature?
This code is copied from `ignore_error` and no modification, so this code tests `ignore_error`.
This code should test 2nd/3rd `<store>`'s `process` is not called.